unstructured/partition/image.py

```diff
@@ -30,16 +39,37 @@ def partition_image(
         A string defining the authentication token for a self-host url, if applicable.
     ocr_languages
         The languages to use for the Tesseract agent. To use a language, you'll first need
-        to isntall the appropriate Tesseract language pack.
+        to install the appropriate Tesseract language pack.
+    strategy
+        The strategy to use for partitioning the PDF. Uses a layout detection model if set
+        to 'hi_res', otherwise partition_image simply extracts the text from the document
+        and processes it.
```

Comment:
Should we document the strategy options?

Reply:
Yep, will document the valid strategy options in here. Also planning a refactor on the strategy handling logic, planned to add a write up on "strategy" to the sphinx docs as part of that.